src/main/java/com/redhat/rhjmc/containerjfr/commands/internal/AbstractRecordingCommand.java

```diff
@@ -90,7 +94,24 @@ protected AbstractRecordingCommand(
             if (ALL_EVENTS_TEMPLATE.getName().equals(templateName)) {
                 return enableAllEvents(connection);
             }
-            return connection.getTemplateService().getEventsByTemplateName(templateName);
+            // if a template name is specified, try to find a Custom template by that name. If none,
+            // fall back on finding a target built-in template by that name. If not, throw an
+            // exception and bail out.
+            return connection
```

Comment:
I guess a side-effect here to keep in mind for the c-jfr-web UI listing of profiles is if a custom profile has the same name as a JVM one, it will make the JVM one no longer accessible with this API.

Reply:
Oh wait... this is the command while the TargetTemplateGetHandler supports specification of template type. Would it make sense to support that here too?

Reply:
Yea, that's something I had in mind to figure out. I don't know how I want to change the `template=Foo` syntax, though. Any suggestions for how to specify custom vs remote template?

Reply:
Looking at the event syntax, comma delimited would be fine.

`template=X,type=Y.`

Or if you want to be more concise, though I guess disallowing commas in the name:
`template=X,Y`

Either seems fine to me. And for ease of use maybe the existing code can be kept for the case where users don't supply a template type.

Reply:
I'll implement the `template=X,type=Y` format. I'll take the shortcut for now and require the keys to be in that order so I can just extend the existing validation regex, but maybe this should use a CSV parser instead... ?

The `template=Name,Type` format looks to me like a list of templates to enable, as if they would be unioned. That's also an interesting feature idea though.

Reply:
Okay, sounds fine. I think restricting order and using regex for now is okay. Re. list of templates, that's a good point. I'm not sure we'd ever want to bother with multiple template input; I don't think JFR/JMC does that with the jfc stuff, does it? But it could be a neat feature. Anyways heh

Reply:
Yea I don't think the unioned template thing is done in JMC, but I could be wrong. Although the union `A U B U C ... U Z` does implicitly happen if you simultaneously create recordings from each template `A...Z` and then retrieve the contents of those recordings, doesn't it? But then you have a bunch of redundant recordings for no good reason.

Reply:
Ah true, I forgot about that. Recordings each have a single jfc but the event configuration in JFR is a union of all the recording event configs, which is in the end a union of the jfc files, heh.